src/import-data/import-utils.ts

```diff
@@ -43,6 +44,11 @@ export const buildDestinationWorkspaceFilter = (
       return false;
     }
 
+    // Do not allow importing data into the non-Google workspace.
+    if (!isGoogleWorkspace(workspace)) {
+      return false;
+    }
+
```

Comment:
this if clause and the following if clause, which checks `getCloudProviderFromWorkspace`, are redundant. I think we should either keep the `requiredCloudPlatform` and eliminate the check on `isGoogleWorkspace`, or keep this check and remove the `requiredCloudPlatform` concept altogether. If we anticipate the current Terra UI to be single-cloud for the foreseeable future (as compared to Manifold being multi-cloud), I'd suggest simplifying by removing `requiredCloudPlatform`.

Reply:
Fixed: https://github.com/DataBiosphere/terra-ui/pull/5275/commits/16fbe74c5f3a23b2a401e5f188c5593c496fc124